docs/install.rst

```diff
@@ -30,12 +30,13 @@ Add `tenant_schemas.routers.TenantSyncRouter` to your `DATABASE_ROUTERS` setting
         'tenant_schemas.routers.TenantSyncRouter',
     )
     
-Add the middleware ``tenant_schemas.middleware.TenantMiddleware`` to the top of ``MIDDLEWARE_CLASSES``, so that each request can be set to use the correct schema.
+Add the middleware ``tenant_schemas.middleware.TenantMiddleware`` or ``tenant_schemas.middleware.SuspiciousTenantMiddleware`` to the top of ``MIDDLEWARE_CLASSES``, so that each request can be set to use the correct schema.
```

Comment:
Add the middleware ``tenant_schemas.middleware.TenantMiddleware`` to the top of ``MIDDLEWARE_CLASSES``, so that each request can be set to use the correct schema. If a tenant is not found, i.e., the requests hostname couldn't be mapped to a particular tenant, an error 404 will be raised. If you'd like to raise ``DisallowedHost`` instead use the middleware ``tenant_schemas.middleware.SuspiciousTenantMiddleware``.